nest/02.introduction.md

```diff
@@ -0,0 +1,51 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all the files required to install, uninstall and update a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage packages**.
+It is similar to an app store for a smartphone, or the Windows Store for example.
+One of the key differences between those and Nest (and most package managers), is that the upgrading must be done manually.
+On a smartphone, the applications are usually automatically upgraded.
+With Nest, it's up to the user to upgrade their installed packages whenever they want.
+
+Currently, to use Nest, one must use its Command Line Interface (CLI).
+This might sound intimidating for someone who has never seen a terminal or typed any command before, but if that's the case, don't worry! This guide is here to help you feel at home.
+
+## 1.2 Packages and dependencies
+It is common for a package to require other ones to work properly.
+For example, let's take a software that allows its users to send messages to each other, over the Internet.
+That software will need a way to send messages over the Internet.
+To do so, it will either implement its own solution, or use an existing library for that purpose.
+In the latter case, the software **requires** that library, because without it, the messages simply can't be sent.
+In other words, that library is a **dependency** of the software.
+
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+If it does, Nest will install the dependencies that aren't already installed.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Within each repository, there are a certain number of categories, and a package has to belong to one, and only one of them.
```

Comment:
```suggestion
Each repository has one or more categories, and each package has to belongs to a specific one of these.
```
(This is phrased as a requirement, not a fact, it may be a good idea to change that too.)

Reply:
you wrote `each package has to belongs`, but I'm guessing you meant `each package belongs`?

Reply:
Woops!
I've edited the comment above.
I wanted to keep it as a requirement since you wrote it like that originally, but I'm fine with this version too.